Server/Quality_of_Life_changer.BLL/Handlers/СommandHandlers/AddUserHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Quality_of_Life_changer.Implication.Handlers.СommandHandlers;
+
+using Contracts.Commands;
+using Data;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Web.Helpers;
+
+public class AddUserHandler : BaseCommandHandler, IRequestHandler<AddUserCommand, AddUserResponse>
```

Comment:
handler naming should correspond to command/query name. if it's handling `AddUserCommand`, it should be `AddUserCommandHandler`

Reply:
also check this in other places